txpool/txpool.go

```diff
@@ -335,7 +335,7 @@ func (p *TxPool) AddTx(tx *types.Transaction) error {
 // ready for execution. (primaries)
 func (p *TxPool) Prepare() {
 	// fetch primary from each account
-	primaries := p.accounts.getPrimaries()
+	primaries := p.accounts.getPrimaries(p.GetBaseFee())
 
 	// create new executables queue with base fee and initial transactions (primaries)
 	p.executables = newPricesQueue(p.GetBaseFee(), primaries)
```

Comment:
Nit: retrieve `baseFee` to local var and reuse it.
```suggestion
    baseFee := p.GetBaseFee()

	primaries := p.accounts.getPrimaries(baseFee)

	// create new executables queue with base fee and initial transactions (primaries)
	p.executables = newPricesQueue(baseFee, primaries)
```